otm-gateway.html

```diff
@@ -223,15 +223,46 @@ <h3><dfn>Deposit Object</dfn></h3>
 
       <section>
         <h3><dfn>Get Object Audit</dfn></h3>
-        <p></p>
+        <p>Describes the status and history of the object's preservation. For more information about auditing, see <a href="audit-appendix.html">OTM Appendix - Audit Events</a>.</p>
+        <p>The information exposed by this endpoint is intended to be informational, allowing repository administrators
+          to monitor content and resolve issues encountered during deposit. The Gateway MUST report any errors that would
+          prevent it from requesting deposit via the Bridge in the <code>gateway-errors</code> field. The Gateway MAY
+          choose to poll status information from the Bridge asynchronously, and is not required to to provide real-time
+          information. The repository SHOULD NOT rely on the information in fields other than <code>gateway-errors</code>
+          to be up-to-date in real time.
+        </p>
         <ul>
           <li>Request Type: <code>GET</code></li>
-          <li>Request Path: <code>/{object-id}/audit</code></li>
+          <li>Request Path: <code>/{object-id}/audit ? versionId=</code></li>
```

Comment:
Is `versionId` required or optional?

Reply:
optional, i think; where omitting it returns data for all versions